scripts/run_dummy_models_on_val.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import os
+from src.dummy_models import (
+    dummy_random,
+    dummy_always_bald,
+    dummy_always_non_bald,
+    evaluate_dummy_model,
+)
+
+val_csv_path = os.path.join("..", "src", "data", "val.csv")
```

Comment:
use dummy_evaluatin_config

Reply:
changed to get_metrics() from evaluation.py instead